fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1962,6 +1997,27 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
 
             if (!enableStorageCache && allowAsyncWriteBack) {
                 throw new DdlException("storage allow_async_write_back can't be enabled when cache is disabled");
+            } else {
+                if (distributionInfo.getBucketNum() == 0) {
+                    int backendNum = GlobalStateMgr.getCurrentSystemInfo().getBackendIds().size();
+                    int bucketNum = 0;
+                    // When POC, the backends is not greater than three most of the time.
+                    // The bucketNum will be given a multiplier factor for small backends.
+                    if (backendNum <= 3) {
```

Comment:
duplicate code with calAvgBucketNumOfRecentPartitions，you can put it in a new funciton or simplify the calculate logic.